backend-rust/src/graphql_api/node_status.rs

```diff
@@ -147,45 +186,116 @@ impl Service {
     }
 }
 
-#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone, Default)]
 #[serde(rename_all = "camelCase")]
 #[derive(SimpleObject)]
 #[graphql(complex)]
-pub struct NodeStatus {
-    pub node_id: String,
-    pub node_name: Option<String>,
+pub struct ExternalNodeStatus {
```

Comment:
I am pretty certain most of these fields requested are not being used by the frontend.

For now lets just support them all.